src/data/mock_astarte.rs

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
This file was renamed to `mock_astarte_sdk.rs` in other PR, delete it.

Reply:
Donte 👍🏻